apps/app/public/mockServiceWorker.js

```
@@ -0,0 +1,344 @@
+/* eslint-disable */
```

ESLint

[error] Specify the rules you want to disable.

(unicorn/no-abusive-eslint-disable)